plugins/hw_event/hw_event_plugin.go

```diff
@@ -15,60 +15,38 @@
 package main
 
 import (
-	"bytes"
-	"crypto/tls"
+	"context"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"os"
 	"sync"
+	"time"
 
 	"github.com/redhat-cne/cloud-event-proxy/pkg/common"
 	hwevent "github.com/redhat-cne/sdk-go/pkg/hwevent"
 
 	"github.com/redhat-cne/sdk-go/pkg/pubsub"
 	"github.com/redhat-cne/sdk-go/pkg/types"
 
+	"github.com/redhat-cne/cloud-event-proxy/plugins/hw_event/pb"
+	"google.golang.org/grpc"
+
 	v1amqp "github.com/redhat-cne/sdk-go/v1/amqp"
 	v1hwevent "github.com/redhat-cne/sdk-go/v1/hwevent"
 	v1pubsub "github.com/redhat-cne/sdk-go/v1/pubsub"
 	log "github.com/sirupsen/logrus"
 )
 
 var (
-	resourceAddress string   = "/hw-event"
-	hwSubURL        string   = "https://10.19.109.249/redfish/v1/EventService/Subscriptions/"
-	hwEventTypes    []string = []string{"StatusChange", "ResourceUpdated", "ResourceAdded", "ResourceRemoved", "Alert"}
-
+	resourceAddress string = "/hw-event"
```

Comment:
do you want to call this as hw-event?
may be the resource should have also node-name, so the subscribe can choose to subscribe to node it is interested of 
/cluster/node/%s/redfish/event
also have ability to subscribe to single HW events , for an example 
/cluster/node/%s/redfish/event/fan

Reply:
done at hw-event-proxy